src/strategies/razor-network-voting/index.ts

```diff
@@ -61,38 +62,38 @@ export async function strategy(
     }
   };
 
-  if (snapshot !== 'latest') {
-    // @ts-ignore
-    params.delegators.__args.block = { number: snapshot };
-  }
-
```

Comment:
This should not be removed @rajkharvar The results should be depended on the block number, else the results can be manuplated

Reply:
Ok. will update that.

Reply:
@ChaituVR I have made the required changes.